torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1478,7 +1494,28 @@ def _make_tensor_helper(shape, low=None, high=None):
                SkipInfo('TestCommon', 'test_variant_consistency_eager',
                         dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16)),),
            sample_inputs_func=sample_inputs_addr),
-
+    OpInfo('amax',
+           op=torch.amax,
```

Comment:
This line is redundant as op is `torch.amax` by default, anyway. This line would be removed in another PR.